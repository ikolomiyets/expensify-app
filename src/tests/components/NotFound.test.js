import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFound';

test('expect not found to be rendered', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});
