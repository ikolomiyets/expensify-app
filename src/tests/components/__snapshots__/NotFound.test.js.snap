// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect not found to be rendered 1`] = `
<div
  className="container"
>
  404! - 
  <Link
    replace={false}
    to="/"
  >
    Go Back
  </Link>
</div>
`;
